test/shared/src/main/scala/zio/test/ZIOSpecAbstract.scala

```diff
@@ -154,15 +153,13 @@ abstract class ZIOSpecAbstract extends ZIOApp with ZIOSpecAbstractVersionSpecifi
         (ZLayer.succeedEnvironment(environment1) ++ liveEnvironment) >>> (TestEnvironment.live ++ ZLayer
           .environment[Scope] ++ ZLayer.environment[ZIOAppArgs])
 
-      eventRenderer           = getTestEventRenderer(testArgs)
-      executionEventSinkLayer = sinkLayer(console, eventRenderer)
```

Comment:
Instead of constructing the layer here, per task, we're just using the layer that was constructed a single time by `ZTestRunnerJVM`